data.json

```diff
@@ -1272,28 +1314,34 @@
     "issueNumber": 648
   },
   {
-    "name": "Harkirat Singh",
-    "githubUsername": "harkiratsm",
-    "imageUrl": "https://user-images.githubusercontent.com/71957674/130051242-25747856-913e-4613-9cc6-7384dc58d752.png",
-    "issueNumber": 645
+    "name": "vonhyou",
+    "githubUsername": "vonhyou",
+    "imageUrl": "https://user-images.githubusercontent.com/29454668/115579668-f8853280-a2f8-11eb-8372-95202883ac86.png",
+    "issueNumber": 451
   },
   {
-    "name": "Anik Chakraborty",
-    "githubUsername": "Anikcb",
-    "imageUrl": "https://user-images.githubusercontent.com/47493387/130086990-1072a097-f8b7-423b-8eeb-b161870b11ea.png",
-    "issueNumber": 657
+    "name": "Yajush Vyas",
+    "githubUsername": "geekyvyas",
+    "imageUrl": "https://user-images.githubusercontent.com/35025375/98467874-c924b280-21fd-11eb-810b-2d17e0342e10.png",
+    "issueNumber": 163
   },
   {
-    "name": "Ahamed Basha N",
-    "githubUsername": "ahamedbasha-n",
-    "imageUrl": "https://user-images.githubusercontent.com/51391473/130546663-9ff92a77-14a1-4004-8d9c-dbc67fb027df.png",
-    "issueNumber": 663
+    "name": "Yash Indane",
+    "githubUsername": "YashIndane",
+    "imageUrl": "https://user-images.githubusercontent.com/53041219/99989860-22353e80-2dd9-11eb-95c5-07dd04eecd7d.png",
+    "issueNumber": 180
   },
   {
-    "name": "Anas Khan",
-    "githubUsername": "anaskhan28",
-    "imageUrl": "https://user-images.githubusercontent.com/87796038/130655483-7fd4dc84-9f0d-412b-acac-68ab178aaf3c.png",
-    "issueNumber": 666
+    "name": "Yash Sehgal",
+    "githubUsername": "yashsehgal",
+    "imageUrl": "https://user-images.githubusercontent.com/62352288/113475867-6aeeb980-9495-11eb-86be-bfa0085a6f48.png",
+    "issueNumber": 429
+  },
+  {
+    "name": "afc163",
```

Comment:
I still don't see this fixed https://github.com/EddieHubCommunity/awesome-github-profiles/pull/684#pullrequestreview-746421054

Reply:
Sorry @Panquesito7 , fixed now.